package-lock.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "iapp-sconification",
+  "lockfileVersion": 3,
+  "requires": true,
+  "packages": {}
+}
```

Comment:
Not sure you really want to commit this generated file? You probably executed `npm install` at the root level of this repo, which is not necessary.